packages/client/src/resources/graphqlApi.ts

```diff
@@ -11,17 +11,25 @@ import {
 } from '@apollo/client'
 import { BatchHttpLink } from '@apollo/link-batch-http'
 import { setContext } from '@apollo/link-context'
-import { store } from '../state'
+import { getAccessToken } from '@essex/msal-interactor'
+import { msalInstance } from './msalInstance'
 
 const setAuthorizationLink = setContext(async (req, prevContext) => {
-	const state = store.getState()
-	if (!state.auth.jwtIdToken) {
+	if (CONFIG.auth.disabled) {
 		return {}
 	} else {
-		return {
-			headers: {
-				Authorization: state.auth.jwtIdToken,
-			},
+		const accounts = msalInstance.getAllAccounts()
+		if (accounts.length) {
+			const accessToken = await getAccessToken(msalInstance, accounts[0], [
```

Comment:
Since we define our identity scopes in our config, I think we should also put the API access scopes in our config as well